docs/libp2p.stream_muxer.yamux.rst

```diff
@@ -0,0 +1,7 @@
+libp2p.stream\_muxer.yamux
+=========================
```

Comment:
Add a `=` so that is matches the `libp2p.stream\_muxer.yamux` length
Like this
```
libp2p.stream\_muxer.yamux
==========================
```

Reply:
Thank you, Luca